src/templates/item/parts/technology.html

```diff
@@ -174,13 +174,11 @@
         </div>
         <div class="flexrow nowrap">
             <label>{{localize "SR5.Wireless"}}</label>
-            <input
-                class="checkbox"
-                type="checkbox"
-                name="system.technology.wireless"
-                {{checked
-                system.technology.wireless}}
-            />
+            {{> "systems/shadowrun5e/dist/templates/common/Select.html"
```

Comment:
There is a {{selectOptions }} helper from Foundry itself. We should either make use of that in the Select.html part (which I didn't know even existed :D) or use it directly here.

Reply:
Looking at the DataModel PR, I don't want to touch the Select.html file since it was renamed